Doc/library/importlib.metadata.rst

```diff
@@ -171,16 +171,18 @@ group.  Read `the setuptools docs
 <https://setuptools.pypa.io/en/latest/userguide/entry_point.html>`_
 for more information on entry points, their definition, and usage.
 
-*Compatibility Note*
-
-The "selectable" entry points were introduced in ``importlib_metadata``
-3.6 and Python 3.10. Prior to those changes, ``entry_points`` accepted
-no parameters and always returned a dictionary of entry points, keyed
-by group. With ``importlib_metadata`` 5.0 and Python 3.12,
-``entry_points`` always returns an ``EntryPoints`` object. See
-`backports.entry_points_selectable <https://pypi.org/project/backports.entry-points-selectable>`_
-for compatibility options.
-
+.. versionchanged::3.12
```

Comment:
Need spaces here for these to render:

```suggestion
.. versionchanged:: 3.12
```

https://cpython-previews--113175.org.readthedocs.build/en/113175/library/importlib.metadata.html#entry-points

Reply:
Thanks. I was inspecting the output and confused I couldn't find these sections. I checked the build logs for warnings and there were none. Thanks for catching the issue. I've rewritten the docs commit.